cmd/image/qcow2ova/ova/ova.go

```diff
@@ -33,15 +35,22 @@ type OVA struct {
 	ImageName, VolumeName string
 	SrcVolumeSize         int64
 	TargetDiskSize        int64
+	OsId                  string
 }
 
 // Render will generate the OVA spec from the template with all the required information like image name, volume name
 // and size
 func Render(imageName, volumeName string, srcVolumeSize int64, targetDiskSize int64) (string, error) {
```

Comment:
can we pass as param instead of using pkg directly into this function?

Reply:
Done.